GiTinder/appsettings.json

```diff
@@ -4,5 +4,8 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"
-}
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "GiTinderContext": "Server=(localdb)\\mssqllocaldb;Database=GiTinderContext-bbcda641-e9ff-4389-960e-f15a994a944e;Trusted_Connection=True;MultipleActiveResultSets=true"
+  }
+}
```

Comment:
@dypher91 new line is missing at the end of the file

Reply:
done